### PR TITLE
Update Index.cshtml

### PR DESCRIPTION
**Pull request by whitezion**
Para added in index by john

### CODE REVIEW

**Review by Beaulawhite — APPROVED** [review]
Good

**Review by Beaulawhite — APPROVED** [review]
Approved

**Review by Beaulawhite — APPROVED** [review]

**Review by Beaulawhite — APPROVED** [review]
Reviewed

File: HJQuiz/Pages/Index.cshtml [hunk 1]
```diff
@@ -8,4 +8,5 @@
     <h1 class="display-4">Heavenly Joy Quiz Program</h1>
     <p>This is Quiz apps.</p>
     <p>Total: 20 members</p>
+<p>soon join</p>
```
Comment (Beaulawhite) [inline HJQuiz/Pages/Index.cshtml:11]: Nice

**Review by Beaulawhite — APPROVED** [review]

**Review by Beaulawhite — CHANGES_REQUESTED** [review]
check this line

**Review by Beaulawhite — COMMENTED** [review]
aspproved

**Review by Beaulawhite — APPROVED** [review]
approved

**Comment by Beaulawhite** [comment]
its not getting approved

**Review by Beaulawhite — APPROVED** [review]
checked

**Review by whitezion — COMMENTED** [review]
ap[proved

**Review by Beaulawhite — APPROVED** [review]
approved

**Pull request merged**